Documentation/custom-scopes-claims-clients.md

```diff
@@ -12,6 +12,7 @@ The following is the exhaustive list of scopes supported by dex:
 | `email` | ID token claims should include the end user's email and if that email was verified by an upstream provider. |
 | `profile` | ID token claims should include the username of the end user. |
 | `groups` | ID token claims should include a list of groups the end user is a member of. |
+| `id_provider` | ID token claims should include the connector ID and the user ID that is returned by the connector. |
```

Comment:
`provider_id`?

Reply:
I've updated the description to say

> ID token claims should include information from the ID provider. The token will contain the connector ID and the user ID assigned at the provider.

Hopefully it makes more sense now?